backend/src/models/student.js

```diff
@@ -0,0 +1,75 @@
+const { Model } = require('objection')
+const tableStudents = 'students'
+
+class Student extends Model {
+  static get tableName () {
+    return tableStudents
+  }
+
+  static get idColumn () {
+    return ['StudentID']
+  }
+
+  fullName () {
+    return `${this.firstName} ${this.lastName}`
+  }
+
+  static get relationMappings () {
+    const NativeLanguage = require('./nativeLanguage')
+    const Status = require('./status')
+    const StatusUpdate = require('./statusUpdate')
+
+    return {
+      nativeLanguage: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: NativeLanguage,
```

Comment:
```suggestion
        modelClass: NativeLanguage.model,
```
Same for the other `modelClass` properties in the other relations. Reason is because when we `require("./nativeLanguage")`, we are getting a whole bunch of things (not just the model), however in `modelClass` we just need the model.

Reply:
Oh i see.... Cool, thank you. 
Have updated all the modelClass properties accordingly, in the latest commit.